setup.py

```diff
@@ -119,7 +119,7 @@ def __getattr__(self, attr):
         vinfo = _version_helper.generate_git_version_info()
     except:
         vinfo = vdummy()
-        vinfo.version = '2.3.12'
+        vinfo.version = '2.3.13'
         vinfo.release = 'True'
```

Comment:
I strongly recommend to also pull in the PR that sets `vinfo.release` to an actual bool.

Reply:
And another PR that fixes a bug introduced in doing so :)

Reply:
So those would be https://github.com/gwastro/pycbc/pull/5033 and https://github.com/gwastro/pycbc/pull/5098.

Reply:
It fixes a small bug in the versioning page of each chunk, and possibly in the version annotation in the GraceDB uploads.

Reply:
Okay, I've included these now, and will merge if tests pass now.